Dockerfile.redist

```diff
@@ -12,30 +12,36 @@ RUN test -z "$(gofmt -l $(find . -type f -name '*.go' -not -path "./vendor/*"))"
 
 # ldflags "-s -w" strips binary
 # ldflags -X injects commit version into binary
+ENV PLATFORM=x86_64
 
 RUN /usr/bin/license-check -path ./ --verbose=false "Alex Ellis" "OpenFaaS Author(s)" \
  && go test $(go list ./... | grep -v /vendor/ | grep -v /template/|grep -v /build/) -cover \
  && VERSION=$(git describe --all --exact-match `git rev-parse HEAD` | grep tags | sed 's/tags\///') \
  && GIT_COMMIT=$(git rev-list -1 HEAD) \
  && CGO_ENABLED=0 GOOS=linux go build --ldflags "-s -w \
         -X github.com/openfaas/faas-cli/version.GitCommit=${GIT_COMMIT} \
-        -X github.com/openfaas/faas-cli/version.Version=${VERSION}" \
+        -X github.com/openfaas/faas-cli/version.Version=${VERSION} \
+        -X github.com/openfaas/faas-cli/platform.Platform=${PLATFORM}" \
         -a -installsuffix cgo -o faas-cli \
  && CGO_ENABLED=0 GOOS=darwin go build --ldflags "-s -w \
         -X github.com/openfaas/faas-cli/version.GitCommit=${GIT_COMMIT} \
-        -X github.com/openfaas/faas-cli/version.Version=${VERSION}" \
+        -X github.com/openfaas/faas-cli/version.Version=${VERSION} \
+        -X github.com/openfaas/faas-cli/platform.Platform=${PLATFORM}" \
         -a -installsuffix cgo -o faas-cli-darwin \
  && CGO_ENABLED=0 GOOS=windows go build --ldflags "-s -w \
         -X github.com/openfaas/faas-cli/version.GitCommit=${GIT_COMMIT} \
-        -X github.com/openfaas/faas-cli/version.Version=${VERSION}" \
+        -X github.com/openfaas/faas-cli/version.Version=${VERSION} \
+        -X github.com/openfaas/faas-cli/platform.Platform=${PLATFORM}" \
         -a -installsuffix cgo -o faas-cli.exe \
  && CGO_ENABLED=0 GOOS=linux GOARCH=arm GOARM=6 go build --ldflags "-s -w \
         -X github.com/openfaas/faas-cli/version.GitCommit=${GIT_COMMIT} \
-        -X github.com/openfaas/faas-cli/version.Version=${VERSION}" \
+        -X github.com/openfaas/faas-cli/version.Version=${VERSION} \
+        -X github.com/openfaas/faas-cli/platform.Platform=${PLATFORM}" \
```

Comment:
I'm guessing that you need to hard-code the value for `PLATFORM` here or put it as a prefix to the `go build` command - otherwise won't the armhf binary have the wrong platform value as defined above?

Reply:
What was your answer to this @viveksyngh ?

Reply:
Now we are hardcoding the platform value for each type of platform like `x86_64 `,  `armhf` and `arm64`